src/AppInstallerCommonCore/Public/AppInstallerErrors.h

```diff
@@ -113,6 +113,7 @@
 #define APPINSTALLER_CLI_ERROR_PIN_ALREADY_EXISTS               ((HRESULT)0x8A150062)
 #define APPINSTALLER_CLI_ERROR_PIN_DOES_NOT_EXIST               ((HRESULT)0x8A150063)
 #define APPINSTALLER_CLI_ERROR_CANNOT_OPEN_PINNING_INDEX        ((HRESULT)0x8A150064)
+#define APPINSTALLER_CLI_ERROR_PACKAGE_HAS_BLOCKING_PIN         ((HRESULT)0x8A150065)

```

Comment:
This error code is not used.

I think this is a general issue with the current implementation of hiding available versions by pin behavior. User will not be notified why a package upgrade is not applicable (they'll see a generic "no applicable upgrade found" for all pin related failures)

We may need more work to support messages like:

Package upgrade skipped due to this blocking pin.. Use --force to override
Package upgrade skipped due to this pinning pin.. Upgrade specifically or use --force
Package upgrade skipped due to gated pin.. Use --force to override

Reply:
Removed the error code and replaced it by a generic error for when a package is pinned (not specific to blocking)